### DISCUSSION TITLE
Issues with current nightly build?

### DISCUSSION THREAD

**Discussion by olimorris** [mrjones2014/legendary.nvim#118] [General]
I keep getting this error with the new nightly build:

```lua
Error detected while processing /Users/Oli/.local/share/nvim/site/pack/packer/start/legendary.nvim/plugin/legendary.lua:
E5113: Error while calling lua chunk: ...pack/packer/start/legendary.nvim/lua/legendary/utils.lua:238: attempt to call field 'nvim_create_user_command' (a nil value) stack traceback:
        ...pack/packer/start/legendary.nvim/lua/legendary/utils.lua:238: in function 'set_command'
        .../pack/packer/start/legendary.nvim/lua/legendary/cmds.lua:99: in function 'func' vim/shared.lua:193: in function 'tbl_map'
        .../pack/packer/start/legendary.nvim/lua/legendary/cmds.lua:98: in function 'bind'
        ...te/pack/packer/start/legendary.nvim/plugin/legendary.lua:6: in main chunk
```

I've rolled back some a couple of versions and a previous legendary commit.

**Comment by mrjones2014** [comment]
The API changed in latest Neovim. If you update to latest nightly it should work.

**Reply by mrjones2014** [reply]
See https://github.com/mrjones2014/legendary.nvim/pull/114

**Reply by olimorris** [reply]
I must have just crossed over with the nightly build!

**Reply by mrjones2014** [reply]
Yeah, I guess I could've handled that better. There shouldn't really be too many breaking changes going forward since Neovim 0.7 will be released pretty soon. If there are breaking changes in the future I'll make a better way to communicate them.

**Reply by olimorris** [reply]
> Yeah, I guess I could've handled that better. There shouldn't really be too many breaking changes going forward since Neovim 0.7 will be released pretty soon. If there are breaking changes in the future I'll make a better way to communicate them. 

Was pretty tricky to foresee. I figured they'd stabilise the api once it's released into the core.

Are you planning on sticking at 0.7 for the minimum version?

**Reply by mrjones2014** [reply]
>Are you planning on sticking at 0.7 for the minimum version?

At least for the moment, it's all I need. I'm just using some APIs added in 0.7. Technically you don't need 0.7 if you only use legendary as a search mechanism and don't have it bind your actual keymaps, but I don't think most people use legendary that way (as far as I know).

We'll have to see what cool features they add to Neovim going forward and see if I can build stuff with them for legendary 😁
